### DISCUSSION TITLE
Snedgrid: payload with metadata

### DISCUSSION THREAD

**Discussion by pkucmus** [saleor/apps#296] [Integrations & features]
Sometimes there's additional business context held in the object's metadata. We might want to have that data available in the context of the template in Sendgrid. So I'm proposing the idea to add the object's metadata to the payload that is being sent to Sendgrid. Maybe there should be a metadata prefix configuration in the plugin's settings to allow one to define that i.e. only SENDGRID. prefixed metadata keys are sent (likely a redundant feature but in case one is very protective of the data there a filter like that could be handy).

**Comment by lkostrowski** [comment]
Duplicate
